src/org/thoughtcrime/securesms/map/MapActivity.java

```diff
@@ -306,7 +306,7 @@ private boolean handleInfoWindowClick(LatLng point) {
     }
 
     private boolean handleAddPoiClick(LatLng point) {
-        if (chatId == 0) {
+        if (chatId == 0 || !DcHelper.getContext(this).getChat(chatId).canSend()) {
```

Comment:
Just something I noticed / FYI: The call to `getChat()` will make clicking on the map to add a POI a little bit slower. But I checked it, `getChat()` can't be avoided here. I mean, it could, but only by loading the chat before the button is clicked, and as we don't want to slow down opening the map either, we would have to do it asynchronously, which wouldn't be worth the effort.